scripts/get_secrets.sh

```diff
@@ -59,20 +59,20 @@ vault kv get -field=license secret/a2/delivery_license | base64 --decode >compon
 
 # Automate acceptence env secrets
 target_host=$(vault kv get -field=data secret/a2/testing/target_host)
-target_username=$(vault kv get -field=data secret/a2/testing/target_username)
-target_password=$(vault kv get -field=data secret/a2/testing/target_password)
+target_user=$(vault kv get -field=data secret/a2/testing/target_user)
+target_key=$(vault kv get -field=data secret/a2/testing/target_key | base64)
```

Comment:
I think you are going to need to upload a base64 encoded version because in CI getting these secrets is handled by buildkite

Reply:
ohhhh, good to know! 👍